docker/entrypoint.sh

```diff
@@ -15,7 +15,10 @@ echo "ZEBRA_CACHED_STATE_DIR=$ZEBRA_CACHED_STATE_DIR"
 echo "LIGHTWALLETD_DATA_DIR=$LIGHTWALLETD_DATA_DIR"
 
 case "$1" in
-    -- | cargo)
+    --* | -*)
```

Comment:
This creates 3 different cases:
- "zebrad -arguments" runs "zebrad -arguments"
- "-arguments" runs "zebrad -arguments"
- "" runs nothing

I wonder if that will be confusing, can we run zebrad with no arguments as well?

```suggestion
    --* | -* | "")
```

Reply:
> * "zebrad -arguments" runs "zebrad -arguments"
> * "-arguments" runs "zebrad -arguments"
> * "" runs nothing

We have to segregate this in two big scenarios:
1. The only image with an `entrypoint.sh` is the test image
2. When running `docker run zfnd/zebra` there's no `entrypoint.sh` and the command (`CMD`) at the end is established as `zebrad -c "${ZEBRA_CONF_PATH}" start` which will be overridden by any other command when running `docker run zfnd/zebra <command>`

From a `docker` perspective we have the following scenarios for the `tests` image:
- `docker run <test_image>`: this will _run nothing_ based on the entrypoint

https://github.com/ZcashFoundation/zebra/pull/5968/files#diff-4f5cabe26761257a4d685a6edc7a43e0fe0f78762f50eeb48530f2bd3b3ee7caR21

- `docker run <test_image> -<argument>` and docker run <test_image> --<argument> will run zebrad + the following arguments

Reply:
Ok, I'm happy with whatever changes you want to make here, or if you just want to leave it as it is.